src/app/views/Home/Components/Pod.elm

```diff
@@ -0,0 +1,68 @@
+module Home.Components.Pod exposing (..)
```

Comment:
Yup, it's exposing all methods. This is actually an anti-pattern I think, I've seen some discussion about it on the elm slack, but this is what is taught in the introduction. Will need to make this more granular